python/cudf/cudf/_lib/groupby.pyx

```diff
@@ -41,15 +41,31 @@ from cudf._lib.cpp.types cimport size_type
 
 # The sets below define the possible aggregations that can be performed on
 # different dtypes. These strings must be elements of the AggregationKind enum.
-_CATEGORICAL_AGGS = {"COUNT", "SIZE", "NUNIQUE", "UNIQUE"}
-_STRING_AGGS = {"COUNT", "SIZE", "MAX", "MIN", "NUNIQUE", "NTH", "COLLECT",
-                "UNIQUE"}
+_CATEGORICAL_AGGS = {"COLLECT", "COUNT", "NUNIQUE", "SIZE", "UNIQUE"}
```

Comment:
Can we sort these in the same order as [their definition in `AggregationKind`](https://github.com/rapidsai/cudf/blob/c5894a1db144c149de0cc6954102936b948de10e/python/cudf/cudf/_lib/aggregation.pyx#L34)? Or alphabetically? Currently I can't see the logic for this order. (It is a set, so it has no meaningful order once constructed, but I'd like to have a rule for how this is written).
```suggestion
_CATEGORICAL_AGGS = {"COUNT", "SIZE", "NUNIQUE", "COLLECT", "UNIQUE"}
```
Likewise for the other sets below.

Reply:
All of the entries are sorted alphabetically?

Reply:
I think I failed to sort “NUNIQUE” by “N” in my brain when assessing if they were alphabetical. Thanks for the correction. 😅